### PR TITLE
Add description of `--overwrite-checkpoint-ts` for changefeed resume

### PR DESCRIPTION
**Pull request by zhaoxinyu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -522,9 +522,12 @@ cdc cli changefeed pause --pd=http://10.0.10.25:2379 --changefeed-id simple-repl
...
 ```
 
-以上命令中：
-
 - `--changefeed-id=uuid` 为需要操作的 `changefeed` ID。
+- `--overwrite-checkpoint-ts`：从 v6.2 开始支持指定 changefeed 恢复的起始 TSO。TiCDC 集群将从这个 TSO 开始拉取数据。该项支持 `now` 或一个具体的 TSO，指定的 TSO 应在 (GC safe point, CurrentTSO] 范围内。
+
+> **注意：**
+> 若 `--overwrite-checkpoint-ts` 指定的 TSO `t2` 大于 changefeed 的当前 checkpoint TSO `t1`（可通过 `cdc cli changefeed query` 命令获取），则会导致 `t1` 与 `t2` 之间的数据不会同步到下游，造成数据丢失。
+> 若 `--overwrite-checkpoint-ts` 指定的 TSO `t2` 小于 changefeed 的当前 checkpoint TSO `t1`，则会导致 TiCDC 集群从一个旧的时间点 `t2` 重新拉取数据，可能会造成数据重复（例如 TiCDC 下游为 MQ sink）。
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:530]:
```suggestion
> - 若 `--overwrite-checkpoint-ts` 指定的 TSO `t2` 大于 changefeed 的当前 checkpoint TSO `t1`（可通过 `cdc cli changefeed query` 命令获取），则会导致 `t1` 与 `t2` 之间的数据不会同步到下游，造成数据丢失。
> - 若 `--overwrite-checkpoint-ts` 指定的 TSO `t2` 小于 changefeed 的当前 checkpoint TSO `t1`，则会导致 TiCDC 集群从一个旧的时间点 `t2` 重新拉取数据，可能会造成数据重复（例如 TiCDC 下游为 MQ sink）。
```

**Comment by zhaoxinyu** [comment]
TODO: 添加 `--no-confirm` 的相关描述

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
@leoppro PTAL. Thanks~

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -522,9 +522,14 @@ cdc cli changefeed pause --pd=http://10.0.10.25:2379 --changefeed-id simple-repl
...
 ```
 
-以上命令中：
-
 - `--changefeed-id=uuid` 为需要操作的 `changefeed` ID。
+- `--overwrite-checkpoint-ts`：从 v6.2 开始支持指定 changefeed 恢复的起始 TSO。TiCDC 集群将从这个 TSO 开始拉取数据。该项支持 `now` 或一个具体的 TSO（如 434873584621453313），指定的 TSO 应在 (GC safe point, CurrentTSO] 范围内。
+- `--no-confirm`：恢复同步任务时无需用户确认相关信息。默认为 false。
+
+> **注意：**
+>
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:530]: 这里是不是可以补一句话：
 `--overwrite-checkpoint-ts` 指定的 TSO `t2` 必须等于 changefeed 的当前 checkpoint TSO `t1`。如果 TSO 不同，会导致数据同步出现问题。

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: ticdc/manage-ticdc.md (hunk 2 above)
Comment (ran-huang) [reply ticdc/manage-ticdc.md:530]: > 指定的 TSO 应在 (GC safe point, CurrentTSO] 范围内。

L526 的最后一句就是这个意思吧。指定的 TSO <= CurrentTSO。可能写得不太清楚？

Comment (hfxsd) [reply ticdc/manage-ticdc.md:530]: 嗯，明确写一下比较好吧

Comment (ran-huang) [reply ticdc/manage-ticdc.md:530]: @zhaoxinyu PTAL

Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:530]: 这个地方必须跟当前 checkpointTs 相等，如果要从当前的 checkpointTs 同步数据，那就没有必要加这个 `--overwrite-checkpoint-ts`，加这个参数的话本身就是用户想从一个特定的位置重新开始同步数据

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -522,9 +522,14 @@ cdc cli changefeed pause --pd=http://10.0.10.25:2379 --changefeed-id simple-repl
 cdc cli changefeed resume --pd=http://10.0.10.25:2379 --changefeed-id simple-replication-task
 ```
 
-以上命令中：
-
 - `--changefeed-id=uuid` 为需要操作的 `changefeed` ID。
+- `--overwrite-checkpoint-ts`：从 v6.2 开始支持指定 changefeed 恢复的起始 TSO。TiCDC 集群将从这个 TSO 开始拉取数据。该项支持 `now` 或一个具体的 TSO（如 434873584621453313），指定的 TSO 应在 (GC safe point, CurrentTSO] 范围内。
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:526]:
```suggestion
- `--overwrite-checkpoint-ts`：从 v6.2 开始支持指定 changefeed 恢复的起始 TSO。TiCDC 集群将从这个 TSO 开始拉取数据。该项支持 `now` 或一个具体的 TSO（如 434873584621453313），指定的 TSO 应在 (GC safe point, CurrentTSO] 范围内。如未指定该参数，默认从当前的 checkpointTs 同步数据。
```

Comment (ran-huang) [reply ticdc/manage-ticdc.md:526]: @zhaoxinyu @hfxsd 这样是否更清楚呢？

Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:526]: 我觉得可以的

**Review by asddongmen — APPROVED** [review]

**Pull request merged**